koku/masu/database/presto_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -51,71 +145,104 @@ WITH cte_ocp_on_azure_resource_id_joined AS (
-        max(ocp.node_capacity_cpu_core_hours) as node_capacity_cpu_core_hours,
-        max(ocp.node_capacity_memory_gigabytes) as node_capacity_memory_gigabytes,
-        max(ocp.node_capacity_memory_gigabyte_hours) as node_capacity_memory_gigabyte_hours,
-        max(ocp.cluster_capacity_cpu_core_hours) as cluster_capacity_cpu_core_hours,
-        max(ocp.cluster_capacity_memory_gigabyte_hours) as cluster_capacity_memory_gigabyte_hours,
-        max(ocp.persistentvolumeclaim) as persistentvolumeclaim,
-        max(ocp.persistentvolume) as persistentvolume,
-        max(ocp.storageclass) as storageclass,
-        max(json_format(volume_labels)) as volume_labels,
-        max(ocp.persistentvolumeclaim_capacity_gigabyte) as persistentvolumeclaim_capacity_gigabyte,
-        max(ocp.persistentvolumeclaim_capacity_gigabyte_months) as persistentvolumeclaim_capacity_gigabyte_months,
-        sum(ocp.volume_request_storage_gigabyte_months) as volume_request_storage_gigabyte_months,
-        sum(ocp.persistentvolumeclaim_usage_gigabyte_months) as persistentvolumeclaim_usage_gigabyte_months
+    max(cast(coalesce(azure.quantity, azure.usagequantity) as decimal(24,9))) as usage_quantity,
+    max(coalesce(billingcurrencycode, currency)) as currency,
+    max(cast(coalesce(azure.costinbillingcurrency, azure.pretaxcost) as decimal(24,9))) as pretax_cost,
+    max(cast(coalesce(azure.costinbillingcurrency, azure.pretaxcost) as decimal(24,9))) * cast({{markup}} as decimal(24,9)) as markup_cost, -- pretax_cost x markup = markup_cost
+    cast(NULL as double) as pod_cost,
+    cast(NULL as double) as project_markup_cost,
+    sum(ocp.pod_usage_cpu_core_hours) as pod_usage_cpu_core_hours,
+    sum(ocp.pod_request_cpu_core_hours) as pod_request_cpu_core_hours,
+    sum(ocp.pod_limit_cpu_core_hours) as pod_limit_cpu_core_hours,
+    sum(ocp.pod_usage_memory_gigabyte_hours) as pod_usage_memory_gigabyte_hours,
+    sum(ocp.pod_request_memory_gigabyte_hours) as pod_request_memory_gigabyte_hours,
+    max(ocp.cluster_capacity_cpu_core_hours) as cluster_capacity_cpu_core_hours,
+    max(ocp.cluster_capacity_memory_gigabyte_hours) as cluster_capacity_memory_gigabyte_hours,
+    max(ocp.pod_labels) as pod_labels,
+    NULL as volume_labels,
+    max(azure.tags) as tags,
+    row_number() OVER (partition by azure.uuid) as project_rank,
+    1 as data_source_rank,
+    max(azure.resource_id_matched) as resource_id_matched
     FROM hive.{{schema | sqlsafe}}.azure_openshift_daily as azure
-    JOIN postgres.{{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary as ocp
-        ON coalesce(azure.date, azure.usagedatetime) = ocp.usage_start
-            AND (
+    JOIN hive.{{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary as ocp
+        on coalesce(azure.date, azure.usagedatetime) = ocp.usage_start
+        -- I kept this AND condition the same, I notice Andrew removed the data source check on the ocp on aws
```

Comment:
What does this comment mean?

Reply:
Oh that was a note to myself, I can. remove that now.